configure.ac

```diff
@@ -262,11 +262,19 @@ ACX_PTHREAD
 
 AC_CHECK_LIB([uuid], [uuid_parse], [true], AC_MSG_FAILURE([libuuid not found]))
 
+# libblkid
+AC_CHECK_HEADER([blkid/blkid.h], [],
+  AC_MSG_ERROR([blkid/blkid.h not found (libblkid-dev, libblkid-devel)]))
+AC_CHECK_LIB([blkid], [blkid_get_cache], [true],
+  AC_MSG_FAILURE([libblkid not found]))
+AC_CHECK_LIB([blkid], [blkid_find_dev_with_tag], [true],
+  AC_MSG_FAILURE([libblkid not found]))
+AC_CHECK_LIB([blkid], [blkid_dev_devname], [true],
+  AC_MSG_FAILURE([libblkid not found]))
```

Comment:
I can do that, but would I need to move the code into a linux-only compiled object (i.e., not FileStore), right? I suppose I can take care of that via your other suggestion down below.